<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UniqLogger DbViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QDateTimeEdit" name="dteStart">
      <property name="displayFormat">
       <string>yyyy-MM-dd HH:mm:ss.zzz</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5" rowspan="3">
     <widget class="QPushButton" name="btnFilter">
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Module</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Severity</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Message body</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>End</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateTimeEdit" name="dteStop">
      <property name="displayFormat">
       <string>yyyy-MM-dd HH:mm:ss.zzz</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="txtModule"/>
    </item>
    <item row="2" column="3">
     <widget class="QComboBox" name="cmbSeverity"/>
    </item>
    <item row="2" column="4">
     <widget class="QLineEdit" name="txtMessage"/>
    </item>
    <item row="3" column="0" colspan="6">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>826</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionChoose_Db_file"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionChoose_Db_file">
   <property name="text">
    <string>Choose Db file...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
